pandas/tests/groupby/aggregate/test_other.py

```diff
@@ -488,12 +488,12 @@ def test_agg_structs_series(structure, expected):
 
 
 @pytest.mark.xfail(reason="GH-18869: agg func not called on empty groups.")
-def test_agg_category_nansum():
+def test_agg_category_nansum(observed):
```

Comment:
Does this need to be xfailed for `observed=True`? I think it may be the right answer (not sure though).

Reply:
fixed so that observed=True is XPASS (not sure how to xfail on a particular fixture value)